misago/users/tests/test_anonymizeoldips.py

```diff
@@ -1,43 +1,45 @@
 import datetime
```

Comment:
Please be explicit when working with `datetime` module, and replace this line with `from datetime import timedelta`.